frontend/src/routes/MediaRating/index.tsx

```diff
@@ -1,70 +1,92 @@
 import { Grid, GridItem } from '@chakra-ui/react'
 import { useEffect, useRef, useState } from 'react'
-import api from '../../api';
-
+import { PostTrialsRequest, PostTrialsRequestTrialMetadata, PostTrialsRequestRatingSet, PostTrialsRequestRatingSetRatingInner, PostTrialsRequestRatingSetRatingInnerEmotion } from "../../schema";
 import MediaDisplay from "../../features/MediaDisplay"
 import RatingSlider from "../../features/RatingSlider"
-import { ListImagesInner } from '../../schema'
 import "./MediaRating.css"
 import { useLocation, useNavigate } from 'react-router-dom'
-
-interface MediaRate {
-    mediaID: number
-    valence: number
-    arousal: number
-}
+import MediaList from "../../../src/imageFileNames.json"
 
 interface RatingPageProps {
     mediaType: string
 }
 
+const detasetName = "OASIS";
+const experimentLocation = "KC111";
+const platform = "";
+const rating_second_by_media = 0;
 const rateSeconds = 5;
```

Comment:
ここ同じ用途の変数やから合わせちゃおうか。
94行目で使ってる`rateSeconds`は`ratingSecondByMedia`に置換しておいてほしい。
あと、JS/TSはキャメルケースで変数名を定義する点に注意してね！
```suggestion
const ratingSecondByMedia = 5;
```